Animal Wellfare and Euthanasia/SOP_BirdMonitoringAndTracking.md

```diff
@@ -0,0 +1,68 @@
+
+The following letter in the bird's name will indicate the ring color. Here are some examples:
+
+- B: Blue
+- BLK: Black
+- LB: Light Blue
+- BR: Brown
+- R: Red
+- W: White
+- Y: Yellow
+- G: Green
+- GR: Gray
+- O: Orange
+
+The next character after the color letter will be the ring number. The number should be an integer (e.g., 1, 4444, 213). Do not start numbering with a zero. For example, use "3" instead of "03" if the ring has the number 03. A number should always appear in the bird's name. If the bird has rings with no number,the bird should be ringed with another ring that contains a number.
+
+### Example Bird Names
+
+- LG4: Left leg green ring number 4.
+- LLB4444: Left leg light blue ring number 4444.
+- RO293: Right leg orange ring number 293.
+- LBRRR2: Left leg brown ring (no number) and right leg red ring number 2.
+- LBLK4RG168: Left leg black ring 4 and right leg green ring 168.
+
+## Monitoring and Tracking Bird History and Current State
+
+Just as individuals have medical histories, birds in our care also possess medical records. In some cases, experiments require the use of the same bird over extended periods, ranging from weeks to months and even years. It is imperative that we meticulously track the history and current state of each bird. This not only constitutes good practice but also serves to minimize any potential harm to the bird, the experiments, or the risk of errors.
+
+### Bird Management Platform
+
+To facilitate this tracking, we utilize the [Bird Management Platform](https://prodis.weizmann.ac.il/pls/htmldb/f?p=468:101:11177740050203:::::). Please note that access to this platform is required to view the provided link.
+
+#### Platform Content
+
+The Bird Management Platform encompasses comprehensive information about all the birds in our lab, including those presently in residence. Each bird's dedicated page includes the following details:
+
+- **Bird's ID (Name)**
+- **Sex and Type:** Canary or Zebra Finch
+- **Arrival Date and Source**
+- **Current Location**
+- **Parents**
+- **Hatch Date**
+- **Age in Post-Hatch Days**
+- **Reservation Status and Details**
+- **Participation in IACUC**
+- **Vital Status (Alive/Deceased) and Surgical History**
+- **Additional Notes**
+- **Files**
+- **Pregnancy Information**
+- **Transfers and Training Details**
+- **Procedures (e.g., Medical Examinations, Virus Injections)**
+- **Bird's Weight**
+
+### Best Practices for Experimenters
+
+Experimenters are required to consult the Bird Management Platform before undertaking any bird-related operation. Additionally, it is crucial to update the bird's page post-operation or any alteration in the bird's current state. This includes ensuring the bird's health, confirming it is not reserved for someone else, and verifying that it is not engaged in other experiments.
```

Comment:
Break this into different bullets.
Currently this bullet deals with several different things.